### ISSUE TITLE
Oral candidiasis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6409]
**Preferred term label:**
Oral candidiasis

**Synonyms**
Oral thrush

**Definition (free text, please give PubMed ID)**
 accumulation and overgrowth of the fungus Candida albicans on the mucous membranes of the mouth, generally manifested as associated with creamy white lesions on the tongue or inner cheeks, occasionally spreading to the gums, tonsils, palate or oropharynx.
PMID:  31424866 

**Parent term (use hpo.jax.org/app)**
Abnormal oral physiology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: # 615527

**Comment by pnrobinson** [comment]
In the HPO a single episode of thrush would be classified as an acute disease. Repeated episodes of thrush are modeled as a phenotype (that can indicate an immunodeficiency). The latter term exists, https://hpo.jax.org/app/browse/term/HP:0009098

**Comment by pnrobinson** [comment]
Closing
